s4e-backend/seed-configs/seed-1/products.json

```diff
@@ -9,7 +9,8 @@
     "granuleArtifactRule": {
       "default": "quicklook",
       "COG": "RGBs_8b"
-    }
+    },
+    "productCategory": 8
```

Comment:
I'd consider using productCategory.name here, you wouldn't need to specify ids in the seeding endpoint